android/src/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -554,7 +557,20 @@ private KiwixWebView newTab() {
 
   private KiwixWebView newTab(String url) {
     KiwixWebView webView = new KiwixWebView(KiwixMobileActivity.this);
-    webView.setWebViewClient(new KiwixWebViewClient(KiwixMobileActivity.this, mLeftArrayAdapter));
+    webView.setWebViewClient(new KiwixWebViewClient(KiwixMobileActivity.this, mLeftArrayAdapter){
+      @Override
+      public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        // onClick
+        if (isFirstRun) {
+          contentsDrawerHint();
+          SharedPreferences.Editor editor = getSharedPreferences("PREFERENCES", MODE_PRIVATE).edit(); 
```

Comment:
Use    SharedPreferences settings = PreferenceManager.getDefaultSharedPreferences(this);
    SharedPreferences.Editor editor = settings.edit();

And in all other locations.
As that is what is used mostly in the app